deck.rb

```diff
@@ -0,0 +1,35 @@
+# The deck should let us remove a card. It should tell us if a card is valid.
+# After removing a card, the card should no longer be valid.
+require './card'
+
+class Deck
+
+	def initialize
+    @cards = []
+    @suits = ["H" , "D", "S", "C"]
+    @values = [1,2,3]
+  
+    @suits.each do |suit| 
+      @values.each do |value|
+        @cards.push(Card.new(suit, value))
+      end
+    end 
+	end
+
+	def validate(card)
+    @suits.include?(card.suit) && @values.include?(card.value)
+	end
+	
+	def remove(card)
+    @cards.delete_if do |cardInDeck| 
+			cardInDeck.suit == card.suit && cardInDeck.value == card.value
+		end
+	end
```

Comment:
Can you clean up the spacing in this file and in card.rb? Lining things up helps a lot with readability. I think most Rubyists like to use 2-space indents: https://github.com/bbatsov/ruby-style-guide#source-code-layout